vars/treebuild.groovy

```diff
@@ -38,17 +38,15 @@ def upstreamBuild(def projectCollection, String currentProject, String settingsX
  * @param skipTests boolean to skip tests or not
  */
 def buildProject(String project, String settingsXmlId, String goals, boolean skipTests, String defaultGroup = "kiegroup") {
-    def projectNameGroup = project.split("\\/")
-    def group = projectNameGroup.size() > 1 ? projectNameGroup[0] : defaultGroup
-    def name = projectNameGroup.size() > 1 ? projectNameGroup[1] : project
+    def group = getProjectGroupName(project, defaultGroup)[0]
+    def name = getProjectGroupName(project, defaultGroup)[1]
```

Comment:
A little bit of nitpicking, feel free to merge if needed as it is: 
I would suggest not to call `getProjectGroupName()` twice (to avoid evaluating it twice), maybe store the result pair in a variable and get it from there?